src/storage/storage.cc

```diff
@@ -88,6 +88,13 @@ Storage::Storage(Config *config)
 Storage::~Storage() {
   DestroyBackup();
   CloseDB();
+  SkipBlockCacheDeallocationOnClose();
```

Comment:
nit: you can change to `TrySkip`, since it may not stop this

Reply:
ok